pyrene/src/components/CheckboxPopover/CheckboxList.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```

Comment:
Should we disable this globally (eslintrc) instead of in every file?

Reply:
We still need this rule for making sure the `public Pyrene components` work well.

For these component we have to `type` the props of the function component. If we do not do that the `babel plugin` cannot generate the `proptype` for `kitchensink`,